source/text/Format.h

```diff
@@ -32,6 +32,9 @@ class Format {
 	// Convert the given number into abbreviated format as described in Format::Credits,
 	// then attach the ' credit' or ' credits' suffix to it.
 	static std::string CreditString(int64_t value);
+	// Creates a string similar to '<amount> tons of <cargo>'.
+	// If there is no specified cargo, the created string is only '<amount> tons'.
+	static std::string MassString(double amount, const std::string &cargo = {});
```

Comment:
Idk if making an empty string like this as a default argument is the best way, honestly
We could just add of smth outside of this method

Reply:
> We could just add of smth outside of this method

What do you mean?

Reply:
You know that when calling the method
Its fine as is, just a bit annoyed we create a string without needing one sometimes

Reply:
This can't be a nullptr, and I want the shorter option. We could add two methods (one with the string, one without), but I think the compiler should optimize this enough for that extra 1 byte of string to not cause a problem.

Reply:
Oh I get what you say. Yeah, I guess that would also work. It's by far the most common use though (iirc every use, except the outfitter), so I thought it would make sense to have a single method for that common code path.